modules/installation-gcp-user-infra-shared-vpc-config-yaml.adoc

```diff
@@ -80,30 +90,31 @@ capabilities:
 ====
 If you disable simultaneous multithreading, ensure that your capacity planning accounts for the dramatically decreased machine performance. Use larger machine types, such as `n1-standard-8`, for your machines if you disable simultaneous multithreading.
 ====
-<5> Specify the main project where the VM instances reside.
-<6> Specify the region that your VPC network is in.
+<5> Optional: A set of network tags to apply to the control plane or compute machine sets. `platform.gcp.defaultMachinePlatform.tags` will apply to both control plane and compute machines. If set, `compute.platform.gcp.tags` and `controlPlane.platform.gcp.tags` will override `platform.gcp.defaultMachinePlatform.tags`.
```

Comment:
```suggestion
<5> Optional: A set of network tags to apply to the control plane or compute machine sets. `platform.gcp.defaultMachinePlatform.tags` will apply to both control plane and compute machines. If set, `compute.platform.gcp.tags` and `controlPlane.platform.gcp.tags` override `platform.gcp.defaultMachinePlatform.tags`.
```

Reply:
You could expand this, too--backticked properties as adjectives to added nouns.

Reply:
Reworked:
<5> Optional: A set of network tags to apply to the control plane or compute machine sets. `platform.gcp.defaultMachinePlatform.tags` applies to both control plane and compute machines. If the `compute.platform.gcp.tags` or `controlPlane.platform.gcp.tags` parameters are set, they override the `platform.gcp.defaultMachinePlatform.tags` parameter.

WDYT?

Reply:
lgtm. could apply to `platform.gcp.defaultMachinePlatform.tags`, too.